resources/views/components/nav-layer.blade.php

```diff
@@ -0,0 +1,57 @@
+@props(['id', 'children', 'title' => __('Menu'), 'hasParent' => false, 'tag' => 'form', 'parentUrl' => ''])
+@slots(['headerbutton'])
+
+<x-rapidez::slideover.mobile
+    :title="(string) $title"
+    :$id
+    :$hasParent
+    :$tag
+>
+    <x-slot:headerbutton>
```

Comment:
i think header-button would be cleaner

Reply:
that would have to to be changed in rapidez/core for the slideover

Reply:
Nevermind then